### DISCUSSION TITLE
Are there ‘call home’ telemetry capabilities on this library?

### DISCUSSION THREAD

**Discussion by srayan** [SixLabors/ImageSharp#2680] [Q&A] [answered]
We are looking to purchase a commercial license and as a part of privacy review we would like to understand if the ImageSharp v3.x library is logging any telemetry that it reports back?

**Comment by tocsoft** [comment, marked as answer]
nope, there is no telemetry or logging in the library. The library never performs network IO at all and only acts on streams provided or some simple disk access for the 2 scenarios or loading and image by file path and saving an image by file path.

As of writing I know of no initiatives that would cause that to change.

**Reply by JimBobSquarePants** [reply]
>As of writing I know of no initiatives that would cause that to change.

There are none.
